GameData/RealismOverhaul/Waterfall_Configs/_Processor/39_glow-resize.cfg

```diff
@@ -0,0 +1,162 @@
+// Resize glow to be the same *visual* size as the main plume. This is necessary because many
+// plumes have "intrinsic" sizes that do not match with glow effects.
+
+@PART[*]:HAS[@ROWaterfall:HAS[#glow]]:BEFORE[zROWaterfall_39_GlowResize]:NEEDS[Waterfall]
+{
+    @ROWaterfall
+    {
+        __glowScale = #$scale$
+
+        // Stretch in z, while accounting for any z-stretching in the main plume itself.
+        // Twice the average scale of x and y by default.
+        @__glowScale[2] = #$scale[0]$
+        @__glowScale[2] += #$scale[1]$
```

Comment:
Sure, I'll reword this. I intentionally avoided using "sum" though, because the logic behind it is "first take the average nozzle diameter, then double it because that's what rocket nozzles are generally shaped like."